.github/workflows/download-pulumi-cron.yml

```diff
@@ -0,0 +1,141 @@
+name: Download Pulumi Cron
+"on":
+  schedule:
+    - cron: 0 8 * * *
+
+jobs:
+  macos-homebrew-install:
+    name: Install Pulumi with Homebrew on macOS
+    runs-on: macos-latest
+    steps:
+      - name: homebrew install
+        run: brew install pulumi
+      - name: Pulumi Version Details
+        id: vars
+        run: |
+          echo ::set-output name=installed-version::$(pulumi version)
+          echo ::set-output name=expected-version::v$(curl -sS https://www.pulumi.com/latest-version)
+      - name: Error if incorrect version found
+        if: steps.vars.outputs.expected-version != steps.vars.outputs.installed-version
+        run: |
+          echo "Expected version ${{ steps.vars.outputs.expected-version }} but found ${{ steps.vars.outputs.installed-version }}"
+          exit 1
+  macOS-direct-install:
+    name: Install Pulumi via script on macOS
+    runs-on: macos-latest
+    steps:
+      - run: curl -fsSL https://get.pulumi.com | sh
+      - run: echo "/Users/runner/.pulumi/bin" >> $GITHUB_PATH
+      - name: Pulumi Version Details
+        id: vars
+        run: |
+          echo ::set-output name=installed-version::$(pulumi version)
+          echo ::set-output name=expected-version::v$(curl -sS https://www.pulumi.com/latest-version)
+      - run: which pulumi
+      - name: Error if incorrect version found
+        if: steps.vars.outputs.expected-version != steps.vars.outputs.installed-version
+        run: |
+          echo "Expected version ${{ steps.vars.outputs.expected-version }} but found ${{ steps.vars.outputs.installed-version }}"
+          exit 1
```

Comment:
Being unfamiliar w/ GHA, is there a way to have this block be re-used across all the jobs?

Reply:
I wish I could do that :/